content/docs/rendering-elements.md

```diff
@@ -18,58 +18,58 @@ const element = <h1>Hola, mundo</h1>;
 
 A diferencia de los elementos del DOM de los navegadores, los elementos de React son objetos planos, y su creación es de bajo costo. React DOM se encarga de actualizar el DOM para igualar los elementos de React.
 
->**Note:**
+>**Nota:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Uno podría confundir los elementos con el muy conocido concepto de "componentes". En la siguiente sección [siguiente sección](/docs/components-and-props.html) hablaremos de componentes. Los elementos son los que "constituyen" los componentes, y recomendamos leer esta sección antes de continuar.
 
-## Rendering an Element into the DOM
+## Renderizando un Elemento en el DOM
 
-Let's say there is a `<div>` somewhere in your HTML file:
+Digamos que hay un `<div>` en alguna parte de tu archivo HTML:
 
 ```html
 <div id="root"></div>
 ```
 
-We call this a "root" DOM node because everything inside it will be managed by React DOM.
+Nosotros lo llamamos un nodo "raíz" porque todo lo que esté dentro de él será manejado por React DOM.
 
-Applications built with just React usually have a single root DOM node. If you are integrating React into an existing app, you may have as many isolated root DOM nodes as you like.
+Las aplicaciones construidas con solo React usualmente tienen un único nodo raíz en el DOM. Dado el caso que estés integrando React en una aplicación existente, podrías tener cuantos nodos raíz en el DOM como quieras.
```

Comment:
Woldn't 

> ... aplicaciones construidas solo con React ...

sound better tan

> ... aplicaciones construidas con solo React ...

And, on the last sentence, this is my suggestion

> Si estás integrando React en una aplicación existente, puedes tener tantos nodos raíz DOM asilados como desees.

Reply:
Well, the original text says "_DOM nodes_" not "_nodes in the DOM_", that's why I wrote "_nodos raíz DOM_". However, what I wrote was wrong too, DOM is an attribute of "node", and "root" an attribute of a "_DOM node"_. So the sentence would finally be:

> Si estás integrando React en una aplicación existente, puedes tener tantos nodos DOM raíz aislados como quieras.

Swapping the order between `raíz` and `DOM`.

Reply:
I suggest 

>  puedes tener tantos nodos` originarios||raiz` del DOM aislados como quieras`

Reply:
@EzequielMonforte That sounds about right. I'd keep using `raíz`, though.

Reply:
Sounds better, I was about to propose that one